packages/date-picker/src/utils/handleDisabledDays.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Modifier,
+  RangeModifier,
+  BeforeAfterModifier,
+} from "react-day-picker/types/Modifiers"
+
+import { DayOfWeek } from "../DatePicker/DatePicker"
+import { daysToNumbers } from "./daysToNumbers"
+
+/**
+ * To save the consumer from passing an array of disabledDay options (each being a different object).
+ * We have split the options out so the consumer can pass in an individual disabledDay option for ease of use.
+ */
+
+export const handleDisabledDays = (
```

Comment:
Somewhat of a nit pick, but I think `handle` is the wrong word here. `calculateDisabledDays`. `determineDisabledDays`, or something like that would make more sense IMO